apollo-api/src/main/java/com/apollographql/apollo/api/Query.java

```diff
@@ -1,5 +1,7 @@
 package com.apollographql.apollo.api;
 
-/** TODO */
+/**
+ * Represents an abstraction for GraphQL query that will be sent to the server.
```

Comment:
I personally prefer comments like ` Represents a custom GraphQL scalar type`  rather than things that mention something is an abstraction.  So this one I think sounds better as "Represents a GraphQL query that will be sent to a server"